docs/pages/custom-builds/schema.mdx

```diff
@@ -740,9 +740,10 @@ You can replace the `eas/build` command call by using these steps in your YAML c
 
 All-in-one function that installs Maestro, prepares a testing environment (Android Emulator or iOS Simulator), and tests the app.
 
-| Input       | Description                                                                                                                                                 |
-| ----------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `flow_path` | **string** Path (or multiple paths, each in a separate line) to [Maestro flows](https://maestro.mobile.dev/getting-started/writing-your-first-flow) to run. |
+| Input       | Description                                                                                                                                                                                                                   |
+| ----------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `flow_path` | **string** Path (or multiple paths, each in a separate line) to [Maestro flows](https://maestro.mobile.dev/getting-started/writing-your-first-flow) to run.                                                                   |
+| `app_path`  | **string** Path (or regex pattern) to the simulator/emulator app that should be tested. If not provided defaults to `ios/build/Build/Products/*simulator/*.app` for iOS and `android/app/build/outputs/**/*.apk` for Android. |
```

Comment:
```suggestion
| `app_path`  | **string** Path (or regex pattern) to the simulator/emulator app that should be tested. If not provided, it defaults to `ios/build/Build/Products/*simulator/*.app` for iOS and `android/app/build/outputs/**/*.apk` for Android. |
```

missing context about "what" defaults to the values. Please also see Simek's comment which provides suggestion on using the correct platform order as per our writing style guide.